timestamp-impl/src/test/java/com/palantir/timestamp/PersistentTimestampServiceMockingTest.java

```diff
@@ -35,23 +30,22 @@
 
     private static final long INITIAL_TIMESTAMP = 12345L;
     private static final long TIMESTAMP = 100 * 1000;
-    private static final TimestampRange SINGLE_TIMESTAMP_RANGE = TimestampRange.createInclusiveRange(TIMESTAMP, TIMESTAMP);
+    private static final TimestampRange SINGLE_TIMESTAMP_RANGE = TimestampRange.createInclusiveRange(TIMESTAMP,
+            TIMESTAMP);
 
     private static final TimestampRange RANGE = TimestampRange.createInclusiveRange(100, 200);
 
     private PersistentTimestamp timestamp = mock(PersistentTimestamp.class);
-    private ExecutorService executor = Executors.newSingleThreadExecutor();
```

Comment:
Nice. I'm surprised this executor still existed at all - but I guess this is why we have baseline

Reply:
This wasn't baseline - I randomly found some unused code while fixing other baseline issues with that file, and followed the [boy scout rule](https://deviq.com/boy-scout-rule/).

Reply:
Ah right. I always thought 'variable that was never used' was a baseline rule (it's probably just one of the intellij checks)